PWGHF/D2H/TableProducer/dataCreatorDV0Reduced.cxx

```diff
@@ -0,0 +1,393 @@
+    hfReducedCollision(collision.posX(), collision.posY(), collision.posZ(),
+                       collision.covXX(), collision.covXY(), collision.covYY(),
+                       collision.covXZ(), collision.covYZ(), collision.covZZ(),
+                       0);
+  }
+
+  void processDsStar2(aod::Collisions const& collisions,
+                      CandsDplusFiltered const& candsDplus,
+                      aod::TrackAssoc const& trackIndices,
+                      aod::V0Datas const& V0s,
+                      aod::BCsWithTimestamps const& bcs)
+  {
+    // handle normalization by the right number of collisions
+    hfCollisionCounter(collisions.tableSize());
+
+    for (const auto& collision : collisions) {
+      auto thisCollId = collision.globalIndex();
+      auto candsDThisColl = candsDplus.sliceBy(candsDplusPerCollision, thisCollId);
+      auto V0sThisColl = V0s.sliceBy(candsV0PerCollision, thisCollId);
+      runDataCreation<DecayChannel::DsStar2ToDplusK0s, CandsDplusFiltered>(collision, candsDThisColl, V0sThisColl, bcs);
+    }
+  }
+  PROCESS_SWITCH(HfDataCreatorDV0Reduced, processDsStar2, "Process DsStar2 to Dplus K0s without MC info and without ML info", true);
+
+  void processDs1(aod::Collisions const& collisions,
+                  CandDstarFiltered const& candsDstar,
+                  aod::TrackAssoc const& trackIndices,
+                  aod::V0Datas const& V0s,
+                  aod::BCsWithTimestamps const& bcs)
+  {
+    // handle normalization by the right number of collisions
+    hfCollisionCounter(collisions.tableSize());
+
+    for (const auto& collision : collisions) {
+      auto thisCollId = collision.globalIndex();
+      auto candsDThisColl = candsDstar.sliceBy(candsDstarPerCollision, thisCollId);
+      auto V0sThisColl = V0s.sliceBy(candsV0PerCollision, thisCollId);
+      runDataCreation<DecayChannel::Ds1ToDstarK0s, CandDstarFiltered>(collision, candsDThisColl, V0sThisColl, bcs);
+    }
+  }
+  PROCESS_SWITCH(HfDataCreatorDV0Reduced, processDs1, "Process Ds1 to DStar K0s without MC info and without ML info", false);
+
+  void processXc(aod::Collisions const& collisions,
+                 CandsDplusFiltered const& candsDplus,
+                 aod::TrackAssoc const& trackIndices,
+                 aod::V0Datas const& V0s,
+                 aod::BCsWithTimestamps const& bcs)
+  {
+    // handle normalization by the right number of collisions
+    hfCollisionCounter(collisions.tableSize());
+
+    for (const auto& collision : collisions) {
+      auto thisCollId = collision.globalIndex();
+      auto candsDThisColl = candsDplus.sliceBy(candsDplusPerCollision, thisCollId);
+      auto V0sThisColl = V0s.sliceBy(candsV0PerCollision, thisCollId);
+      runDataCreation<DecayChannel::Xc3055ToDplusLambda, CandsDplusFiltered>(collision, candsDThisColl, V0sThisColl, bcs);
+    }
+  }
+  PROCESS_SWITCH(HfDataCreatorDV0Reduced, processXc, "Process Xc to Dplus Lambda without MC info and without ML info", false);
```

Comment:
This one should be merged with the DsStar2 (and maybe you can rename it `DplusV0`)